pandas/core/frame.py

```diff
@@ -5717,6 +5720,16 @@ class    max    type
         monkey         mammal    NaN    jump
         """
         inplace = validate_bool_kwarg(inplace, "inplace")
+        if inplace is not no_default:
+            warnings.warn(
+                "'inplace' will be removed in a future version "
+                "and the current default behaviour ('inplace=False') will "
+                "be used. Set 'inplace=False' to silence this warning.",
```

Comment:
This won't work after chaning to no_default. Something llike Remove inplace to silence this warning?